rest/src/main/kotlin/builder/message/modify/MessageModifyBuilder.kt

```diff
@@ -35,6 +38,18 @@ public sealed interface MessageModifyBuilder {
 
     public var attachments: MutableList<DiscordAttachment>?
 
+    /**
+     * Optional custom [MessageFlags] to add to the message created.
+     *
+     * @see suppressEmbeds
+     */
+    public val flags: MessageFlags?
```

Comment:
is this intended to be a `val` too? (i.e. is there a reason why `flags` shouldn't be set to whatever a user wants?)

Reply:
after reviewing the rest of this pr, i'm pretty sure this is supposed to be a var, please change it